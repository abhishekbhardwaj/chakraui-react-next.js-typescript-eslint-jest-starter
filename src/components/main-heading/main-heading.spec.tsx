import React from 'react'
import { render } from '@testing-library/react'

import MainHeading from './main-heading'

describe('Index Page', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<MainHeading />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders hello world', () => {
    // Given
    const { getByTestId } = render(<MainHeading>Hello World</MainHeading>)

    // When
    const ACTUAL = getByTestId('main-heading').textContent

    // Then
    expect(ACTUAL).toEqual('Hello World')
  })
})
