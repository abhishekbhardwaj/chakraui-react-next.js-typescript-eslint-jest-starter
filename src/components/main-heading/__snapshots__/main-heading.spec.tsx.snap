// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="chakra-heading css-0"
    data-testid="main-heading"
  />
</DocumentFragment>
`;
